curator/Dockerfile

```diff
@@ -13,42 +13,37 @@ ENV HOME=/opt/app-root/src \
     CURATOR_ACTIONS_FILE=/etc/curator/settings/actions.yaml \
     CURATOR_LOG_LEVEL=ERROR \
     CURATOR_SCRIPT_LOG_LEVEL=INFO \
-    CURATOR_VER=5.2.0 \
     CURATOR_TIMEOUT=300 \
+    CURATOR_VER=5.8.1 \
     container=oci
 
-ARG LOCAL_REPO
-RUN if [ -n "${LOCAL_REPO}" ] ; then \
-     curl -s -o /etc/yum.repos.d/local.repo ${LOCAL_REPO} ; \
-    fi
+LABEL io.k8s.description="Curator elasticsearch container for elasticsearch deletion/archival" \
+  io.k8s.display-name="Curator ${CURATOR_VER}" \
+  io.openshift.tags="logging,elk,elasticsearch,curator"
```

Comment:
Does shrinking the labels here comply with https://github.com/projectatomic/ContainerApplicationGenericLabels#overview?

Reply:
These are the only ones that will get built into the image using the current art process: `Import description, tags, display-name from the labels in their Dockerfile` https://mojo.redhat.com/docs/DOC-1182562